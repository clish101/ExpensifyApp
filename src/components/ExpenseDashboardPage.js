import React from 'react';
import ExpenseListFilters from './ExpenseListFilters';
import ExpensesSummary from './ExpensesSummary';
import Expenses_list from './Expenses_list';

const ExpenseDashboardPage = () => (
  <div>
    This is from my dashboard component
    <ExpensesSummary />
    <ExpenseListFilters />
    <Expenses_list />
  </div>
);

export default ExpenseDashboardPage;
