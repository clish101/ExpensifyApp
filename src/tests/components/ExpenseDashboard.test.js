import ExpenseDashboardPage from '../../components/ExpenseDashboardPage';
import { shallow } from 'enzyme';
import React from 'react';

test('Should show the dashboard page of the expenses', () => {
  const wrapper = shallow(<ExpenseDashboardPage />);
  expect(wrapper).toMatchSnapshot();
});
