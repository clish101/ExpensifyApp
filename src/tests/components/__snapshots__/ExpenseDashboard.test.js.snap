// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should show the dashboard page of the expenses 1`] = `
<div>
  This is from my dashboard component
  <Connect(ExpensesSummary) />
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</div>
`;
